### PR TITLE
Change Key model scheme.

### PR DESCRIPTION
**Pull request by lyw0149**
Change Key model scheme.

change ambiguous variable name.

@SarahhhC @KyuaKwon @nlpenguin @ethkim @MechanicKim @ifndefdeadmau5 @ace0625

### CODE REVIEW

File: src/server/routes/chat.route.js [hunk 1]
```diff
@@ -1,15 +1,12 @@
...
-    'MESSAGE',
-    `${req.body.sender.name} : ${req.body.message}`
-  );
-  res.end();
-});
+//get message object from Sendbird server and send push notificaiton.
+router.post('/callback', chat.callback);
+
+//get Sendbird appKey for access in client.
+router.get('/appkey', chat.getAppKey);
```
Comment (SarahhhC) [inline src/server/routes/chat.route.js]: Where do we use this?

**Comment by SarahhhC** [comment]
Good! LGTM 👍

File: src/server/controllers/chat.controller.js [hunk 2]
```diff
@@ -0,0 +1,24 @@
+import * as pushUtil from '../utils/push.util';
+import mongoose from 'mongoose';
```
Comment (MechanicKim) [inline src/server/controllers/chat.controller.js]: As Airbnb style guide, absolute import must precede the relative import.

File: src/server/utils/push.util.js [hunk 3]
```diff
@@ -44,19 +44,22 @@ export function sendPush(receiverId, notificationType, bodyParam) {
           },
           priority: 'high',
         };
-        const fcm = new FCM(serverKey);
-        fcm.send(message);
+        const fcm = new FCM(fcmToken);
+        fcm.send(message)
+          .catch((err) => {
+            console.log(err);
```
Comment (MechanicKim) [inline src/server/utils/push.util.js]: What is that console?
It is necessary to leave behind?

File: src/server/controllers/chat.controller.js (hunk 2 above)
Comment (KyuaKwon) [inline src/server/controllers/chat.controller.js]: @MechanicKim but this is not a file. importing a module

File: src/server/utils/push.util.js (hunk 3 above)
Comment (lyw0149) [inline src/server/utils/push.util.js]: It will be replace to custom log

**Pull request merged**